orchagent/mirrororch.cpp

```diff
@@ -758,72 +785,56 @@ void MirrorOrch::updateFdb(const FdbUpdate& update)
 void MirrorOrch::updateLagMember(const LagMemberUpdate& update)
 {
     SWSS_LOG_ENTER();
-    auto sessionIter = m_syncdMirrors.begin();
-    for (; sessionIter != m_syncdMirrors.end(); ++sessionIter)
-    {
-        if (!sessionIter->second.neighborInfo.resolved)
-        {
-            continue;
-        }
 
-        if (sessionIter->second.neighborInfo.port.m_type != Port::LAG)
-        {
-            continue;
-        }
+    for (auto it = m_syncdMirrors.begin(); it != m_syncdMirrors.end(); it++)
+    {
+        const auto& name = it->first;
+        auto& session = it->second;
 
-        // It is possible to have few session that points to one LAG member
-        if (sessionIter->second.neighborInfo.port != update.lag || sessionIter->second.neighborInfo.portId != update.member.m_port_id)
+        // Check the following two conditions:
+        // 1) the neighbor is LAG
+        // 2) the neighbor LAG matches the update LAG
+        if (session.neighborInfo.port.m_type != Port::LAG ||
+                session.neighborInfo.port != update.lag)
```

Comment:
>eighborInfo.port != update.lag [](start = 25, length = 30)

similar to vlan case, we only need to deal with lag member remove case. Even when we add new lag member, we do not need to change existing monitor port.

Reply:
I have thought about this. The issue is that it might be easier to debug if the behavior of always sending to the first member of the LAG to be consistent. That is something different from VLAN where we cannot send the packet to random members. I am fine if we don't do anything for the LAG member add case, which makes the code simpler.